universal-application-tool-0.0.1/app/views/ViewUtils.java

```diff
@@ -28,6 +28,14 @@ public Tag makeLocalJsTag(String filename) {
         .withType("text/javascript");
   }
 
+  /**
+   * Generates a script tag for loading a javascript asset that is provided by a web JAR and found
+   * at the given asset route.
+   */
+  public Tag makeWebJarsTag(String assetsRoute) {
+    return script().withSrc(assetsFinder.path(assetsRoute));
+  }
```

Comment:
praise: nice! i had tried to get this figured out but was missing this piece i think